objects/src/transaction/tx_args.rs

```diff
@@ -78,17 +79,17 @@ impl TransactionArgs {
         self.note_args.get(&note_id)
     }
 
-    /// Returns a reference to the args [AdviceMap].
-    pub fn advice_map(&self) -> &AdviceMap {
-        &self.advice_map
+    /// Returns a reference to the args [AdviceInputs].
+    pub fn advice_inputs(&self) -> &AdviceInputs {
+        &self.advice_inputs
     }
 
     // STATE MUTATORS
     // --------------------------------------------------------------------------------------------
```

Comment:
I would also add `extend_merkle_store()` method which could look something like this:
```rust
pub fn extend_merkle_store<I: Iterator<Item = InnerNodeInfo>>(&mut self, iter: I)
```